### PR TITLE
test: define ALLPERMS if not yet

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: alpine/APKBUILD.in [hunk 1]
```diff
@@ -58,7 +58,6 @@ makedepends="
 	readline-dev
 	rpcgen
 	snappy-dev
-	userspace-rcu-dev
```
Comment (tchaikov) [inline alpine/APKBUILD.in]: @dx9 shall we drop this from this repo? as i think you've submitted the recipe to aport. see https://git.alpinelinux.org/aports/tree/community/ceph .

Comment (tchaikov) [reply alpine/APKBUILD.in]: @iggy Hi Iggy, i found you were updating Ceph's aport recently. could you help review this change?

File: src/test/libcephfs/test.cc [hunk 2]
```diff
@@ -36,6 +36,10 @@
 #include <vector>
 #include <thread>
 
+#ifndef ALLPERMS
+#define ALLPERMS (S_ISUID|S_ISGID|S_ISVTX|S_IRWXU|S_IRWXG|S_IRWXO)
```
Comment (runsisi) [inline src/test/libcephfs/test.cc]: to build under alpine, `ACCESSPERMS` should be defined in `common_init.cc` too

Comment (tchaikov) [reply src/test/libcephfs/test.cc]: why? this macro is not referenced out there, i think.

File: alpine/APKBUILD.in (hunk 1 above)
Comment (dx9) [reply alpine/APKBUILD.in]: @tchaikov OK with me

Comment (tchaikov) [reply alpine/APKBUILD.in]: @dx9 thanks!

Comment (tchaikov) [reply alpine/APKBUILD.in]: will remove alpine packaging in a follow-up PR.

**Pull request merged**